packages/bundle-source/src/index.js

```diff
@@ -547,7 +545,7 @@ ${sourceMap}`;
   throw Error(`unrecognized moduleFormat ${moduleFormat}`);
 }
 
-/** @type {BundleSource} */
+/** @type {import('./types.js').BundleSource} */
```

Comment:
I know I'm in the minority on this but I'd really be fine with,
```suggestion
/** @type {import('./types').BundleSource} */
```

Reply:
I think I see your point. TypeScript is treating all the files with the same base name as a compilation unit.

Reply:
I used `import('./types')` in eventual-send, so I'm amenable to doing it here as well... and done.